README.md

```diff
@@ -0,0 +1,9 @@
+# API Spec Format Converter API
+
```

Comment:
Can you add a line about what this project is and why are we doing it? @pcgamer1

Reply:
Also, for now, it's okay for the README to be so bland, but as we progress, please continue to update it with better instructions and possibly a better format, please. Not on priority as of now
Refer: 
https://github.com/othneildrew/Best-README-Template or any other template for that matter.

Reply:
@meenakshi-dhanani I've added a template based on the above suggestion.